native/nativescript/README.md

```diff
@@ -1,10 +1,32 @@
 # NativeScript
 
-To avoid Out Of Memory exception:
+***To avoid Out Of Memory exception:***
 1- Try when possible to use small images size.
 2- Don't use ImageSource or Base64 encoded string as possible because the Bitmap is transferred to Javascript 
 (Javascript garbage collection happens less frequently than Java garbage collection which might lead to Out Of Memory).
 
+***Steps to solve keyboard is covering screen on IOS:***
+***Note:*** We are trying to imitate how android behaive and working(Scroll into focused input field and keep it on screen).
+So to do that in IOS we have to do the following steps.
+*In your UI (HTML/XML) file:
+1- Add scroll element as a main parent ```<ScrollView #scrollView>``` and give it #id.
+2- Give each input field #id.
+3- Define ```returnKeyType``` property for each input field and gibe them proper value like ```returnKeyType = "next" or returnKeyType = "done"```.
+4- Define (returnPress) callback method for each input fields like ```(returnPress)="goToNextField($event)"```.
+
+*In your style sheet file:
+1- Give that scroll element full size. width,height:100%.
```

Comment:
Can we write this a bit more visually? Something like:
`Give that scroll element full size:`
```
width: 100%
height: 100%
```
? What do you think?

Reply:
It's much better like that, I'll do that sure.